deRSE23/README.md

```diff
@@ -0,0 +1,11 @@
+# deRSE23 Submission
+
+Abstract (and slides) for the [deRSE23](https://de-rse23.sciencesconf.org/) conference.
+
+* Title: Introducing the Fortran-lang community
+* Presenter: Ivan Pribec
+* Co-authors: Vincent Magnin, Asdrubal Lozada-Blanco, Ondrej Certik, Jeremie Vandenplas, Milan Curcic
+
+## Abstract
+
+A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering, and evolution of the language itself, attracting new users and retaining existing ones has been a challenge for Fortran. To address this, an open-source community Fortran-lang has formed in 2019. Here, we present the efforts of Fortran-lang such as the Fortran package manager, standard library, website, LFortran compiler and other tools. Fortran-lang aims to provide a central place for Fortran users, complementing the language standardization and compiler communities. Building on the best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.
```

Comment:
```suggestion
A thriving community is vital for the success and continuity of a programming language. Despite Fortran’s role in science and engineering, and in-spite of the evolution of the language itself, attracting new users and retaining existing ones is challenging. To address this, the open-source community Fortran-lang was formed in 2019. Here, we present the efforts of Fortran-lang including the Fortran package manager, standard library, website, LFortran compiler and other tools. Fortran-lang aims to provide a central home for all Fortran users, complementing the language standardization and compiler communities. Building on best practices in software development, we aim to reduce the language learning curve and promote Fortran adoption.
```
This is now 742 characters, with minor changes for style and consistency :)

Reply:
Thanks Rohit. I would not like to dwell too much over the use of past simple, past participle, etc. Indeed 2019 feels like long ago, but I think "has" in this context, serves to connect the action with the present.

I've taken Sebastian's suggestion to smoothen the use of _despite_ in the second sentence. I also kind of like "challenge for Fortran" at the stress position in the end. Let me know if the latest version looks good to you.